### PR TITLE
Home page stylized

### PR DESCRIPTION
**Pull request by Dielone**
1) Create beautiful Sticky footer.
2) Stylized home page.

### CODE REVIEW

**Comment by Dielone** [comment]
Small fix - changed:

home.html.slim
```
  = link_to("Login",new_user_session_path, class: 'large button')
  = link_to("Sign up",new_user_registration_path, class: 'large button')
```

File: app/assets/stylesheets/core/_home.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#home_page{
```
Comment (vladimirbazhanov) [inline app/assets/stylesheets/core/_home.scss]: pls add spaces between words and { }

File: app/assets/stylesheets/core/all.scss [hunk 2]
```diff
@@ -10,3 +10,4 @@
 @import "base";
 @import "content";
 @import "layout";
+@import "home";
```
Comment (vladimirbazhanov) [inline app/assets/stylesheets/core/all.scss]: we like when import are sorted in alphabetical order

File: app/views/layouts/application.html.slim [hunk 3]
```diff
@@ -19,11 +19,13 @@ html class="no-js" lang="en"
     == analytics_init if GA.tracker
 
   body
-    #root
-      = render "navigation"
-      = render "messages"
+    div id="wrap"
```
Comment (vladimirbazhanov) [inline app/views/layouts/application.html.slim]: in SLIM

div id="wrap"

is the same as

#wrap

File: app/views/layouts/application.html.slim [hunk 4]
```diff
@@ -19,11 +19,13 @@ html class="no-js" lang="en"
...
+      #root
+        = render "navigation"
+        = render "messages"
 
-      = yield
+        = yield
+        div id="push"
```
Comment (vladimirbazhanov) [inline app/views/layouts/application.html.slim]: and here too

File: app/views/pages/home.html.slim [hunk 5]
```diff
@@ -1 +1,10 @@
-= image_tag "book_logo.png"
+div class="row"
+ div id="home_page"
+  div class="small-6 columns"
+   = image_tag "book_logo.png"
+  div class="small-6 columns" id="home_welcome_text"
```
Comment (vladimirbazhanov) [inline app/views/pages/home.html.slim]: div class="small-6 columns" id="home_welcome_text"

.small-6.columns#home_welcome_text

File: app/views/pages/home.html.slim [hunk 6]
```diff
@@ -1 +1,10 @@
...
+   h1 Hello, dear guest
+   H3 Welcome to our web-project
+   .expanded.button-group
+      = link_to("Login",new_user_registration_path, class: 'large button')
```
Comment (vladimirbazhanov) [inline app/views/pages/home.html.slim]: without ( ) will work the same :)

**Comment by vladimirbazhanov** [comment]
🆗
and fix issues pls)

**Pull request closed without merging**